filebeat/files/filebeat.jinja

```diff
@@ -1,73 +1,27 @@
-{%- set filebeat = salt['pillar.get']('filebeat', {}) %}
+# Example configuration file
```

Comment:
My main concern here now is that we aren't templatizing the file, or allowing for it. What was the reasoning behind this modification? Does this have the potential to negatively impact people who might use this formula and expect values to be set in specific locations?